pytest_plugins/metadata_markers.py

```diff
@@ -119,12 +119,15 @@ def pytest_collection_modifyitems(items, config):
         markers_prop_data = []
         exclude_markers = ['parametrize', 'skipif', 'usefixtures', 'skip_if_not_set']
         for marker in item.iter_markers():
-            prop = marker.name
-            if prop in exclude_markers:
+            proprty = marker.name
```

Comment:
typo here.

Reply:
Its deliberately done as `property` is a reserved keyword in python :)

Reply:
Variable could be of any name though, including typing mistakes !

Reply:
`PEP8` suggests to append a trailing underscore rather than introducing a spelling error in such cases

https://peps.python.org/pep-0008/#descriptive-naming-styles
```
...
single_trailing_underscore_: used by convention to avoid conflicts with Python keyword
```

Reply:
OOPs! Its merged !